testutils/build.gradle

```diff
@@ -124,9 +124,16 @@ project.afterEvaluate{
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-
-    implementation project(':keyvault')
-    implementation project(':labapi')
+    if(findProject(":keyVault") != null) {
+        implementation project(path: ':keyvault')
+    } else {
+        implementation project(":keyvault")
+    }
+    if(findProject(":keyVault") != null) {
+        implementation project(path: ':labapi')
+    } else {
+        implementation project(':labapi')
+    }
```

Comment:
What is this logic for? OneAuth?

Reply:
Yes.  I'm amazed it worked - let me try removing it, since it seems to be completely wrong.